client/packages/common/src/intl/locales/en/app.json

```diff
@@ -3,7 +3,6 @@
   "app": "mSupply",
   "auth.alert-title": "Authentication Error",
   "auth.no-store-assigned": "You are not assigned to any stores. Please login as a different user, or contact your administrator.",
-  "auth.permission-denied": "Permission denied",
   "auth.server-error": "The server has returned an error when fetching user details!\nUnable to continue, please contact support@msupply.foundation.",
```

Comment:
I guess this was relocated ? Was wondering why ? 
Also what about all of the translations ?

I think this sort of change is best to be avoided, since our translations not entirely type safe, and quite a lot of work to try and test

Reply:
It's moved to common, and every translation instant falls back to common. Moved it since it's going to be used in more than one place

Reply:
> Also what about all of the translations ?

> It's moved to common, and every translation instant falls back to common

Does this translation still work (english lives in common.json, french lives in app.json): 

https://github.com/msupply-foundation/open-msupply/blob/3c0b740d4fa65120375546e74e19e7caa05c667d/client/packages/common/src/intl/locales/fr/app.json#L40

And if we add it again and use 'common' not the 'app' would french translation still work ?

Reply:
Have moved all the permission-denied [72fbf0a](https://github.com/msupply-foundation/open-msupply/pull/2496/commits/72fbf0ac71055464ed1fad9e8f0ac653e81aa85b)